exchanges/margin/margin.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // Valid returns whether the margin type is valid
 func (t Type) Valid() bool {
-	return t != Unset && supported&t == t
+	return t != Unset && (t&(t-1) == 0) && supported&t == t
```

Comment:
`(t&(t-1) == 0)` is fine but it blocks any future combinations and will need to remove if we introduce that for any reason.